plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,250 @@
+      required: true
+      type: str
+    secure:
+      description:
+        - ssl connection.
+      type: bool
+      default: false
+    timeout:
+      description:
+        - timeout of request nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - skip validation cert.
+      type: bool
+      default: true
+    cert:
+      description:
+        - path of certificate ssl.
+      type: path
+    key:
+      description:
+        - path of certificate key ssl.
+      type: path
+    namespace:
+      description:
+        - namespace for nomad.
+      type: str
+    token:
+      description:
+        - acl token for authentification.
+      type: str
+    name:
+      description:
+        - name of job for delete or stop job without source.
+      type: str
+    state:
+      description:
+        - type of request
+      choices: ["present", "absent"]
+      required: True
+      type: str
+    force_start:
+      description:
+        - force job to started
+      type: bool
+      default: false
+    source_json:
+      description:
+        - source of json nomad job
+      type: json
+    source_hcl:
+      description:
+        - source of hcl nomad job
+      type: str
+'''
+
+EXAMPLES = '''
+- name: create job
+  nomad_job:
+    host: localhost
+    state: present
+    source_hcl: "{{ lookup('file', 'job.hcl') }}"
+    timeout: 120
+
+- name: stop job
+  nomad_job:
+    host: localhost
+    state: absent
+    name: api
+
+- name: force job to start
+  nomad_job:
+    host: localhost
+    state: present
+    name: api
+    timeout: 120
+    force_start: true
+'''
+
+
+import os
+import json
+
+from ansible.module_utils.basic import AnsibleModule
+
+try:
+    import nomad
+    import_nomad = True
+except ImportError:
+    import_nomad = False
+
+
+def run():
+    module = AnsibleModule(
+        argument_spec=dict(
+            host=dict(required=True, type='str'),
+            state=dict(required=True, choices=['present', 'absent']),
+            secure=dict(type='bool', default=False),
+            timeout=dict(type='int', default=5),
+            validate_certs=dict(type='bool', default=True),
+            cert=dict(type='path', default=None),
+            key=dict(type='path', default=None),
+            namespace=dict(type='str', default=None),
+            name=dict(type='str', default=None),
+            source_json=dict(type='json', default=None),
+            source_hcl=dict(type='str', default=None),
+            force_start=dict(type='bool', default=False),
+            token=dict(type='str', default=None, no_log=True)
+        ),
+        mutually_exclusive=[
+            ["name", "source_json", "source_hcl"],
+            ["source_json", "source_hcl"]
+        ],
+        required_one_of=[
+            ['name', 'source_json', 'source_hcl']
+        ]
+    )
+
+    if not import_nomad:
+        module.fail_json(msg='python-nomad is required for nomad_job ')
```

Comment:
Use `missing_required_lib` from `ansible.module_utils.basic` to format this message.